da-tinker/questioner
# [#163139981] Implement functionality for GET /meetups/upcoming endpoint

**What does this PR do?**
Implement functionality for the GET /meetups/upcoming endpoint

**Description of Task to be completed**
1. Write tests
2. Write code to pass tests
3. Test endpoint with [Postman](https://www.getpostman.com/)

**How should be manually tested?**
1. Clone the repository
```bash
$ git clone https://github.com/da-tinker/questioner.git
```
2. Navigate to project folder
```bash
    $ cd questioner
```
3. Ensure you're on the get all meetups branch
```bash
$ git checkout ft-endpoint_get_all_meetups-163139981
```
4. Activate the virtual environment
```bash
$ source env/bin/activate
```
5. Install requirements
```bash
$ pip install -r requirements.txt
```
6. Run the app
```bash
$ flask run
```
7. Run tests
```bash
$ pytest 
```

**Screenshots**
![possible_bug_list_is_list_fail_but_should_pass](https://user-images.githubusercontent.com/45802848/51083703-4e88ca00-172f-11e9-9856-46cc6369f15a.png)
![possible_bug_list_is_list_fail_but_should_pass_pdb](https://user-images.githubusercontent.com/45802848/51083704-4f216080-172f-11e9-8caa-5fcb592afbe2.png)
![possible_bug_list_is_list_pass_but_should_fail](https://user-images.githubusercontent.com/45802848/51083705-4f216080-172f-11e9-859d-8d34689a394f.png)
![postman_test_get_all_meetups_pass](https://user-images.githubusercontent.com/45802848/51083706-4fb9f700-172f-11e9-8121-36d98814f39d.png)
![test_all_passing](https://user-images.githubusercontent.com/45802848/51083707-4fb9f700-172f-11e9-9e9f-87576e778b9a.png)
![test_storage_get_all_meetups_fail_1](https://user-images.githubusercontent.com/45802848/51083710-4fb9f700-172f-11e9-8a0e-b61312228115.png)

**What are the relevant Pivotal Tracker Story IDs?**
[#163139981](https://www.pivotaltracker.com/story/show/163139978)
